package.json

```diff
@@ -46,5 +31,26 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.1",
+    "@fullcalendar/core": "^5.4.0",
+    "@fullcalendar/daygrid": "^5.4.0",
+    "@fullcalendar/interaction": "^5.4.0",
+    "@fullcalendar/react": "^5.4.0",
+    "@fullcalendar/timegrid": "^5.4.0",
+    "@material-ui/core": "^4.11.2",
```

Comment:
Die Dependencies kann man nicht einfach so ohne weiteres in die devDependencies schieben :sweat_smile: 
z.B. Fullcalendar und material-ui werden sicher auch im Produktiven Code (mit `build`) verwendet

Reply:
Ups... Habe ich korrigiert resp. zurück in die Dependencies geschoben